File: java/src/main/java/pickles/PickleTag.java
package pickles;

public class PickleTag {
    private final String name;
    private final PickleLocation source;

    public PickleTag(String name, PickleLocation source) {
        this.name = name;
        this.source = source;
    }

    public String getName() {
        return name;
    }

    public PickleLocation getSource() {
        return source;
    }
}

File: dotnet/Gherkin/Pickles/PickleTag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Gherkin.Pickles
{
    public class PickleTag
    {
        public PickleLocation Location { get; private set; }
        public string Name { get; private set; }

        public PickleTag(PickleLocation location, string name)
        {
            Location = location;
            Name = name;
        }
    }
}
